source/opt/inline_pass.h

```diff
@@ -169,6 +166,52 @@ class InlinePass : public Pass {
   std::unordered_set<uint32_t> funcs_called_from_continue_;
 
  private:
+  // DebugInlinedAtContext keeps data used for generating DebugInlinedAt
+  // instructions.
```

Comment:
Try to make this more meaningful.  What data is kept?  Also add comments describing the functions that are not getters and setters.